fluent_bit/manifest.json

```diff
@@ -0,0 +1,35 @@
+{
+  "display_name": "Fluent Bit",
+  "maintainer": "zhenyamzk+GitHub@gmail.com",
+  "manifest_version": "1.0.0",
+  "name": "fluent_bit",
+  "metric_prefix": "fluentbit.",
+  "metric_to_check": "fluentbit.input.records.count",
+  "creates_events": false,
+  "short_description": "Collect Fluent Bit internal metrics for each running plugin.",
+  "guid": "d64eb215-b1ef-4594-ba17-13bcac668962",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Fluent Bit",
```

Comment:
```suggestion
  "public_title": "Datadog-Fluent Bit (Agent) Integration",
```

Reply:
This is the required format for integrations public_title parameters, and will appear on the page as Fluent Bit (Agent)

Reply:
Thanks. I noticed that not all integrations follow this format.